test/suites/static_analysis.sh

```diff
@@ -47,6 +47,12 @@ test_static_analysis() {
       done
     fi
 
+    OUT=$(godeps . ./shared | cut -f1)
+    if [ "${OUT}" != "$(printf "github.com/gorilla/websocket\ngopkg.in/yaml.v2\n")" ]; then
+      echo "ERROR: you added a new dependency to the client or shared; please make sure this is what you want"
+      echo "${OUT}"
+    fi
```

Comment:
This chunk should be conditional on godeps being around or it will break test environments that are more limited (like Ubuntu's autopkgtest).